### ISSUE TITLE
Fails for remote methods

### ISSUE THREAD

**Issue by kennethlynne** [strongloop/loopback-sdk-angular#159]
```js

  ETUser.prototype.__get__spaces = function (cb) {
    var user = this;
    var SpaceUser = loopback.getModel('SpaceUser');

    SpaceUser.find({
      where: {
        userId: user.id
      },
      include: 'space'
    }, cb);

  };

  ETUser.remoteMethod(
    '__get__spaces',
    {
      isStatic: false,
      description: 'Get the users spaces',
      http: {path: '/spaces', verb: 'get'},
      returns: {arg: 'spaces', type: 'array', root: true}
    }
  );

```

Crashes `lb-ng`

```bash
Loading LoopBack app "...api/server/server.js"
Generating "SDK" for the API endpoint "http://localhost:1337/api"
/usr/local/Cellar/nvm/0.24.1/versions/node/v0.12.2/lib/node_modules/strongloop/node_modules/loopback-sdk-angular-cli/node_modules/loopback-sdk-angular/lib/services.js:106
  var targetClass = modelPrototype[scopeName]._targetClass;
                                             ^
TypeError: Cannot read property '_targetClass' of undefined
    at buildScopeMethod (/usr/local/Cellar/nvm/0.24.1/versions/node/v0.12.2/lib/node_modules/strongloop/node_modules/loopback-sdk-angular-cli/node_modules/loopbac
    -sdk-angular/lib/services.js:106:46)
    at /usr/local/Cellar/nvm/0.24.1/versions/node/v0.12.2/lib/node_modules/strongloop/node_modules/loopback-sdk-angular-cli/node_modules/loopback-sdk-angular/lib/services.js:90:5
    at Array.forEach (native)

```

**Comment by csvan** [comment]
Having a similar issue (https://github.com/strongloop/loopback-sdk-angular/issues/162) though in my case the remote method binding simply isn't generated at all, but lb-ng still terminates normally.

**Comment by bajtos** [comment]
The fix is available in `v1.4.1`.
